chatwoot/chatwoot
# fix: Send message with "enter" also do new line

# Pull Request Template

Fixes https://github.com/chatwoot/chatwoot/issues/5622

## Type of change


## How Has This Been Tested?

1. Enable enter to send messages in profile settings.
2. And write "something" to test in rich editor.
3. And move the cursor to "someth | ing" and press enter key.
4. See the sent message.
5. Now it will be "someth ing" before it will break the line "someth
                                                                                                   ing"

## Checklist:
